### ISSUE TITLE
Cannot setup, 404 on call to auth/alexamedia/proxyverify

### ISSUE THREAD

**Issue by mtherien** [custom-components/alexa_media_player#2101]
**Describe the bug**

After authenticating with Amazon, and entering the Captcha, it is redirected. to a auth/alexamedia/proxyverify which gives a 404 response.

**To Reproduce**
1. In settings, click add integration
2. Select Alexa Media Player
3. Enter email address, amazon password, and 2FA App Key (the URL given is correct
4. Click submit
5. Entered OTP code at Amazon's site
6. Click check box for OTP from the Built-in 2FA App Key confirmed successfully
7. Click submit
8. Presented with Alexa sign on screen with username and password filled in
9. Clicked Sign in
10. Recaptcha appears, enter value, click continue
11. Get a 404 error going to /auth/alexamedia/proxyverify
12. Going back to the tab to add the authentication, an Open Website button appears taking you through the process again
13. Closed the dialog

**Expected behavior**

The Alexa Media Player to appear in the integrations.

**Screenshots**

![image](https://github.com/custom-components/alexa_media_player/assets/246436/eeee789d-fd59-4067-b1dc-321870356f7b)

**System details**

- Home-assistant (version): 2023.10.5 (Docker image)
- alexa_media (version from `const.py` or HA startup): 4.7.7
- alexapy (version from `pip show alexapy` or HA startup): ??
- Amazon [2FA is enabled](https://github.com/custom-components/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account) (y/n). yes

**Logs**
Please provide [logs](https://github.com/custom-components/alexa_media_player/wiki/FAQ#how-do-i-enable-debug-logging-for-the-component).

Can't provide, doesn't install

**Additional context**

**Comment by fdellutri** [comment]
Same issue

**Comment by w0nk1** [comment]
Same!

**Comment by yourbabyscan** [comment]
same issue here

**Comment by ThWelte** [comment]
Same Error

**Comment by rgrdit** [comment]
Same error

**Comment by witc-git** [comment]
Same issue, tried local http and https with Nabu Casa.

**Comment by altheus** [comment]
Same issue - I solved looking at the issue looking at this issue - https://github.com/custom-components/alexa_media_player/issues/2089 

Using @Fredo70's branch of AlexaPy seems to get things working. I ran this code from an SSH Terminal

```
wget https://gitlab.com/Fredo70/alexapy/-/archive/dev/alexapy-dev.zip
unzip alexapy-dev.zip
mv alexapy-dev/alexapy config/
rm -rf alexapy-dev*
ha core restart
```

**Comment by alandtse** [comment]
Please provide logs per the logs instruction via editing configuration.yaml. There's a path that isn't being translated properly and the logs will show it.

**Comment by mtherien** [comment]
After upgrading to the latest version, it appears to be working

**Comment by yourbabyscan** [comment]
I updated from v4.7.7 to v4.7.9 via HACS and still not change for me, still getting 404 after logging into Amazon and completing captcha. Just removed and going to try dev instead to see if any difference for me.

**Comment by witc-git** [comment]
Issue persists after update for me too.

**Comment by yourbabyscan** [comment]
Tried re-installing using dev version, and tried using the oldest available version, both give me the 404: Not Found page at auth/alexamedia/proxyverify after captcha. So does trying with the oldest available version via HACS (v4.7.5)... I figured if it was a bug introduced in a newer version then using an older version may not have the bug, but same result.

Also tried using amazon.com (the default) instead of amazon.co.uk, but again, same 404: Not Found page. :(
